spec/rubocop/ast/keyword_splat_node_spec.rb

```diff
@@ -39,6 +39,82 @@
     it { expect(kwsplat_node.operator).to eq('**') }
   end
 
+  context 'when forwarded keyword rest arguments', :ruby32 do
+    let(:kwsplat_node) { parse_source(source).ast.children.last.children.last }
+
+    describe '.new' do
+      let(:source) do
+        <<~RUBY
+          def foo(**)
+            { a: 1, ** }
+          end
+        RUBY
+      end
```

Comment:
Could we share the `let(:source)`? It would also be closer to the `let(:kwsplat_node)`

Reply:
Ah, makes sense! I've updated it. Thank you for the suggestion!